apps/impala/src/impala/api.py

```diff
@@ -142,20 +144,27 @@ def alanize(request):
     doc = Document2.objects.get(id=query_id)
     snippets = doc.data_dict.get('snippets', [])
     secret = snippets[0]['result']['handle']['secret']
-    impala_query_id = unpack_guid(base64.decodestring(secret))
+    impala_query_id = unpack_guid(decodebytes(secret))
```

Comment:
Have you tested this with Impala editor? And see if you can the query profile.

Reply:
Yes, tested Impala editor's basic functionalities and query profile also. Checking the same thing for new feature what Sreenath have implemented.
<img width="1266" alt="Screenshot 2023-04-12 at 11 27 02 AM" src="https://user-images.githubusercontent.com/36241930/231367083-143b51e3-d095-423b-af5a-160241187655.png">